app/views/components/person_list.scala.html

```diff
@@ -14,24 +14,44 @@
  * limitations under the License.
  *@
 
+@import config.FrontendAppConfig
 @import viewmodels.Person
-@(people: Seq[Person])(implicit messages: Messages)
+
+@(appConfig: FrontendAppConfig, peopleWithFlag: Seq[(Person, Boolean)])(implicit messages: Messages)
 
 <div class="form-group">
     <ul id="people" class="govuk-check-your-answers form-group cya-questions-long">
-        @for(person <- people) {
+        @for((person, flag) <- peopleWithFlag) {
         <li>
             <div class="cya-question">
                 <h2 id="@person.id" class="bold std-text">@person.name</h2>
             </div>
 
-            <div class="cya-change">
+            @if(appConfig.completeFlagEnabled){
+
+                @if(flag){
+
+                <div class="cya-status status">
```

Comment:
`<div class="cya-status status"> </div>` can be set in the outter scope of `if(flag)`

Reply:
Better to hide an empty column when toggle is off and only show the populated column if the toggle is switched on? Stylesheets may make a visible difference